App/screens/CredentialOffer.tsx

```diff
@@ -2,7 +2,8 @@ import type { RouteProp } from '@react-navigation/native'
 import type { StackNavigationProp } from '@react-navigation/stack'
 import type { HomeStackParams } from 'navigators/HomeStack'
 
-import { CredentialState } from '@aries-framework/core'
+import { ConnectionRecord, CredentialRecord, CredentialState } from '@aries-framework/core'
+import { Metadata } from '@aries-framework/core/build/storage/Metadata'
```

Comment:
@blu3beri can you make sure this gets exported?

Reply:
I may not need this if I can access the `schemaId` using the getter